ColorCode.UWP/Common/VisualHelpers.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace ColorCode.Common
+{
+    public static class VisualHelpers
+    {
+        /// <summary>
+        /// Commandbars causing crashes at the mo
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="parent"></param>
+        /// <param name="deepscan"></param>
+        /// <param name="includeCommandBars"></param>
+        /// <returns></returns>
+        public static T FirstChildofType<T>(DependencyObject parent, bool deepscan = false, bool includeCommandBars = false)
```

Comment:
This looks like just a combo of the LogicalTree Helpers and the VisualTree Helpers already in the toolkit, no?

Reply:
I couldn't see any in the toolkit? Point me?

Reply:
https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI/Extensions/Tree/LogicalTree.cs